### ISSUE TITLE
Disable logging

### ISSUE THREAD

**Issue by JayChandler** [NeovaHealth/t4xmlrpc#3]
Hi

Is there a way to turn of logging ?

Thx

**Comment by biphasic** [comment]
Hello,
what sort of logging backend do you use? According to https://github.com/typesafehub/scala-logging it is a wrapper for the SLF4J library, so you could disable it with a simple .properties file maybe?

**Comment by JayChandler** [comment]
I am using SLF4J with Log4J implementation

**Comment by biphasic** [comment]
Then set your logging level to OFF instead of DEBUG, INFO,... in a log4j.properties file

**Comment by JayChandler** [comment]
Yeah i know this but i want to keep my log messages from my app. Just want to exclude those from the lib :)

**Comment by biphasic** [comment]
log4j.category.com.tactix4.t4xmlrpc=OFF
Does that do the trick?
